src/account.js

```diff
@@ -0,0 +1,23 @@
+
+/* Account login state manager
+ */
+
+import _ from 'lodash'
+import db from './db'
+import { storeKey } from './config'
+
+export default { setUser }
```

Comment:
`setUser` is the only thing exported... but you use it like `account.setUser`... are you expecting to need to export more functions or can this be a `export default setUser`?